paper/main_sections/clustering.tex

```diff
@@ -56,3 +56,47 @@ \subsection{Hierarchical Clustering}
 to see the parts of the brain that have the strongest relationship with the
 signal and compare them to the results of the origin research paper.
 
+\subsection{t-Statisics Grouping}
+
+\par We use the t-statistics that we found earlier for another type of cutoff
+analysis. The t-statistics measure the size of the difference relative to the 
+variation in the data. Although the t-statistics go hand in hand with the 
+Benjamini-Hochberg analysis on the ``p-values'', we ultimately decided that an 
+additional process of selecting t-statistics based on a threshold was 
+necessary because the ``p-values'' assume a normal distribution while the 
+t-statistics do not assume this from the data.
+
+\par Since the magnitude from zero represents how significant the test is for
+t-statistics, we collected the absolute value t-statistics that were above a 
+certain threshold. This was the opposite of the procedure for finding the 
+significant tests in the Benjamini-Hochberg process. The threshold, also called
+the cutoff in the scripts that were written when developing this process, is 
+calculated in a very similar fashion to the Benjamini-Hochberg false discovery 
+rate. This is because the t-values and the ``p-values'' are strongly linked,
+because lower ``p-values'' and higher t-values are associated with significant 
+findings. 
+
+\par Implementing the t-value grouping function was almost trivial because it 
+is the flip side of the Benjamini-Hochberg function that was implemented 
+previously.
+
+\subsection{Beta Grouping}
+
+\par Using the same function that we created to group a certain subset of the 
+t-statistics, we also looked over the beta-values as a part of our analysis. 
+Much of what we know in statistics tells us that observing beta-values on top 
+of looking at t-statistics would not tell us much since the two variables are 
+not explicitly connected; however, the main reason why we decided it was of 
+utmost importance to look at both the t-statistics and the beta values is that 
+there might be a relationship between the trends in each respective variables'
+output to the cutoff point. 
+
+\par Granted, it is pretty clear that the t-statistics grouping is very similar
```

Comment:
Intuitively it's mathematically similar (due to the 1-to-1 functional relationship), but it is not the case that we can generalize the similarity (I realized this when playing the last few days)- especially since the parameter Q vs the proportion parameter act in different ways.